app/common/SessionValues.scala

```diff
@@ -21,4 +21,5 @@ object SessionValues {
   val CLIENT_NINO = "ClientNino"
 
   val TAX_YEAR = "TAX_YEAR"
+  val EMPLOYMENT_ID = ""
```

Comment:
could this be updated to have the name be the same as the value and be more explicit that it's a temporary employment id in session. 

So could be val TEMP_NEW_EMPLOYMENT_ID = "TEMP_NEW_EMPLOYMENT_ID"